bigtable-hbase-integration-tests/pom.xml

```diff
@@ -98,6 +152,10 @@ limitations under the License.
                     <artifactId>${google.bigtable.project.under.test}</artifactId>
                     <version>${project.version}</version>
                 </dependency>
+              <dependency>
```

Comment:
I don't think so.  We want to test to make sure that both alpn-boot and tcnative work with NoKnownGap tests.  The Gap test is an orthogonal concern.